### DISCUSSION TITLE
Consistent Marketing card pattern & package card for Console

### DISCUSSION THREAD

**Discussion by kkenna3** [twilio-labs/paste#3889] [Office hours] [answered]
### Background

Voice & Messaging teams both have use cases for upselling products, blocking premium features, and signing up for packaged plans. Upsell pattern we've made is primarily for highlighting related products that would enhance the user's current experience, or a placeholder for a premium feature embedded on a non-premium page. We also wanted a consistent background color, image & CTA placement, and sizing in order to distinguish this card but also have it consistent across products.
We created a template and guidelines based on the marketing card guidelines to have a consistent pattern in Console.

### Feedback

We'd like feedback on the design & guidelines and recommended next steps to best socialize (contributing to patterns?) 

### Stage of design

High fidelity

### Relevant links

Figma has the customer journey, use cases, examples, guidelines, and component: https://www.figma.com/file/LB0ipfVz91IdkA3KRf4KWr/Product-Upsell-%26-Premium-feature-adoption-journey?type=design&t=IwdqT0FVycXb5Q5W-6 

### Are you looking for visual design support?

- [ ] Yes

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours @kkenna3! Here's the [recording](https://twilio.zoom.us/rec/share/U4Pz7ejTFogKXVsXSzXTH5LH0qM1TdrY5Mp9-2mlER4voXwkOuLL0rSa7lLdGiQM.NjAY6F1CYI41Gyw5). You can find the password in slack.
